### PR TITLE
ci: tweak test workflow

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -87,8 +112,16 @@ jobs:
...
+    - name: Run Tests
+      run: |
+        # Configure install options
+        export KAMEL_INSTALL_BUILD_PUBLISH_STRATEGY=Spectrum
+        export KAMEL_INSTALL_REGISTRY=$KIND_REGISTRY
+        export KAMEL_INSTALL_REGISTRY_INSECURE=true
+
         echo "Running tests"
-        yaks test ./test
+        yaks run --logs --wait ./test
```
Comment (nicolaferraro) [inline .github/workflows/test.yaml:123]: Does not seem to wait...
```suggestion
        yaks test ./test
```

Comment (lburgazzoli) [reply .github/workflows/test.yaml]: yeah, it was an attempt to see if for some reason the cli was not working as expected
btw, yaks test does not seems to exists as a sub-command is that correct ?

**Comment by lburgazzoli** [comment]
@nicolaferraro don't know if it is worth merging it, it was mainly an attempt to understand why it was not working.

**Comment by oscerd** [comment]
Do we want to merge this one?

**Comment by lburgazzoli** [comment]
don't think it is needed anymore

**Pull request closed without merging**